mavros/src/lib/uas_data.cpp

```diff
@@ -40,13 +47,15 @@ UAS::UAS() :
 	fcu_capabilities(0)
 {
 	try {
-		// Using smakkest dataset with 5' grid,
+		// Using smallest dataset with 5' grid,
 		// From default location,
 		// Use cubic interpolation, Thread safe
 		egm96_5 = std::make_shared<GeographicLib::Geoid>("egm96-5", "", true, true);
 	}
 	catch (const std::exception &e) {
+		// catch exception and shutdown node
 		ROS_ERROR_STREAM("UAS: GeographicLib exception: " << e.what());
+		signal(SIGINT, term);
 	}
```

Comment:
Well I wasn't sure what was the best approach. I know that SIGINT can be intercepted besised allowing proper debugging. I was thinking that we could added the info on the SIGINT stating that the problem is due to not having the dataset installed. Though I suppose that can also be handled on the ROS_ERROR_STREAM above.